foundations/html_css/flexbox/flexbox-intro.md

```diff
@@ -64,9 +64,7 @@ As you've seen, flexbox is not just a single CSS property but a whole toolbox of
 
 <span id="flex-container-item-knowledge-check">A flex container is any element that has `display: flex` on it. A flex item is any element that lives directly inside of a flex container.</span>
 
-<span id="how-to-create-flex-item-knowledge-check">![container-vs-child](https://cdn.statically.io/gh/TheOdinProject/curriculum/b2a53579fcbec1cfde47646cc5a2b109cd7772cc/foundations/html_css/flexbox/imgs/03.png)
-
-</span>
+<span id="how-to-create-flex-item-knowledge-check">![container-vs-child](https://cdn.statically.io/gh/TheOdinProject/curriculum/b2a53579fcbec1cfde47646cc5a2b109cd7772cc/foundations/html_css/flexbox/imgs/03.png)</span>
```

Comment:
Good spot, thanks for bringing this up. Since this is an image and the span is purely to have an element with an id for a knowledge check link to scroll to, it would actually be better for us to attach the id to the image itself, which can be done in markdown as follows.

```suggestion
![container-vs-child](https://cdn.statically.io/gh/TheOdinProject/curriculum/b2a53579fcbec1cfde47646cc5a2b109cd7772cc/foundations/html_css/flexbox/imgs/03.png){: #how-to-create-flex-item-knowledge-check}
```

A note for future PRs, you're currently PRing from your fork's `main` branch. While this won't cause problems for this PR, it can cause a lot of headaches if you start working on multiple things, with unrelated commits bleeding into different work. You will want to keep your main branch untouched, other than keeping it in sync with upstream (TOP's repo). All your work should go in a different branch coming off `main` for each individual PR, meaning each PR will have its own commit history, untouched by any other branches' work (which is what we want).

You can look through the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) for instructions on the workflow.

Reply:
Thank you for the suggestions! I didn't know about the markdown syntax in your change, I was mainly concerned with removing the unnecessary text, but it's a nice improvement. I guess there's many more features in markdown I don't know about.